python/dbos-cron-starter/html/app.html

```diff
@@ -0,0 +1,143 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="icon"
+      href="https://dbos-blog-posts.s3.us-west-1.amazonaws.com/live-demo/favicon.ico"
+      type="image/x-icon"
+    />
+    <script src="https://cdn.tailwindcss.com"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.9.0/styles/github.min.css" />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.9.0/highlight.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.9.0/languages/python.min.js"></script>
+    <title>DBOS Toolbox</title>
+    <style>
+      .hljs {
+        /* Add these two lines to the existing .hljs rule */
+        white-space: pre-wrap !important;
+        word-wrap: break-word !important;
+      }
+
+      /* Add these two new rules */
+      pre {
+        white-space: pre-wrap !important;
+        word-wrap: break-word !important;
+      }
+
+      pre code {
+        white-space: pre-wrap !important;
+        word-wrap: break-word !important;
+      }
+
+      .hljs {
+        background: #f6f8fa;
+        padding: 0.75em 0.25em 0.75em 0.75em;
+        font-size: 14px;
+        line-height: 1.5;
+      }
+      .step-highlight {
+        background-color: #fff3b4;
+        display: block;
+      }
+    </style>
+  </head>
+  <body class="bg-gray-100 min-h-screen font-sans">
+    <div class="max-w-2xl mx-auto py-12 px-4">
+      <div class="bg-white rounded-lg shadow-lg p-8 space-y-8">
+        <h1 class="text-3xl font-bold text-gray-900">Welcome to the DBOS Cron Starter!</h1>
+        <p class="text-gray-800">
+          DBOS lets you easily run code on a schedule in the cloud. For example, this app runs the following function
+          every minute, updating an in-memory counter:
+        </p>
+
+        <div class="bg-gray-100 rounded-lg overflow-hidden">
+          <pre><code class="python">@DBOS.scheduled("* * * * *")
+@DBOS.step()
+def scheduled_function(scheduled_time, actual_time):
+    global counter
+    counter += 1
+    DBOS.logger.info(f"At {scheduled_time}, incremented counter to {counter}")</code></pre>
+        </div>
+
+        <p class="text-gray-800" id="counter-display">
+            The current value of the counter is <span id="counter-value" class="font-medium">0</span>.
```

Comment:
Would be nice to bold the counter, making it obvious that this value is updating.